frontend/javascripts/libs/utils.js

```diff
@@ -453,7 +475,11 @@ export function addEventListenerWithDelegation(
       }
     }
   };
-  element.addEventListener(eventName, wrapperFunc, false);
+  element.addEventListener(
+    eventName,
+    wrapperFunc,
+    arePassiveEventsSupported() ? { passive: false } : false,
```

Comment:
Now every event listener attached with `addEventListenerWithDelegation` is passive, shouldn't we only do this for `wheel` events? I'd suggest adding another optional parameter `options` which is the same as the parameter to `addEventListener`. When attaching the `wheel` handler we can set passive to false there.